crates/sui-replay-2/src/replay_txn.rs

```diff
@@ -290,6 +331,106 @@ impl ReplayTransaction {
     pub fn checkpoint(&self) -> u64 {
         self.checkpoint
     }
+
+    pub fn digest(&self) -> &TransactionDigest {
+        &self.digest
+    }
+
+    // Get `InputObjects` from a set of (ObjectId, version) pairs, where version is a u64.
```

Comment:
Nit: more code motion that seems a little unnecessary.

Reply:
I moved `get_input_objects_for_replay` to be a function in `ReplayTransaction` given it was using a bunch of field of that struct and it felt more natural. 
That is the reason for all code motion 
(I made that comment above and just copy/pasta here, lol)